src/rules/html/warn-on-remote-script.js

```diff
@@ -0,0 +1,25 @@
+import { VALIDATION_WARNING } from 'const';
+import { isStrictRelativeUrl } from 'schema/formats';
+import * as messages from 'messages';
+
+
+export function warnOnRemoteScript($, filename) {
+  return new Promise((resolve) => {
+    var linterMessages = [];
+
+    $('script').each((i, element) => {
+      const src = $(element).attr('src');
+
+      if (src !== undefined && !isStrictRelativeUrl(src)) {
+        linterMessages.push(
+          Object.assign({}, messages.REMOTE_SCRIPT, {
+            type: VALIDATION_WARNING,
```

Comment:
summoning @jvillalobos for a final statement here.

I do agree with @wagnerand, this should be an ERROR since that's what our policies say. On the other hand, that'd mean that inremove scripts can be removed as a risk-factor for post-review.